### PR TITLE
Change serialization syntax

### PR DESCRIPTION
**Pull request by okkez**
string as following

In previous version:
   string_param1 string_value
   string_param2 string
value

In this version:
   string_param1 "string_value"
   string_param2 "string\nvalue"

I don't change serialization format for other types.

See #522

### CODE REVIEW

File: lib/fluent/config/element.rb [hunk 1]
```diff
@@ -144,6 +142,42 @@ def secret_param?(key)
...
+        param_key = key.to_sym
+        proxy = @corresponding_proxies.detect do |_proxy|
+          _proxy.params.has_key?(param_key)
+        end
+        return nil unless proxy
+        _block, opts = proxy.params[param_key]
+        opts[:type]
+      end
+
+      def dump_value(k, v, indent, nindent)
```
Comment (tagomoris) [inline lib/fluent/config/element.rb]: Is there any reason to assign `out = ""` at once and then do `out << "..."`?
It looks good enough just to return string value by `if` expression.

Comment (okkez) [inline lib/fluent/config/element.rb]: Exactly.
I will fix this.

**Comment by okkez** [comment]
I've fixed and pushed.

**Comment by tagomoris** [comment]
Merged! Thank you so much.

**Pull request merged**